tracker/tracker/src/main/modules/input.ts

```diff
@@ -173,18 +157,60 @@ export default function (app: App, opts: Partial<Options>): void {
       // TODO: support multiple select (?): use selectedOptions; Need send target?
       if (hasTag(node, 'select')) {
         sendInputValue(id, node)
-        app.attachEventListener(node, 'change', () => {
+        const handler = () => {
           sendInputValue(id, node)
-        })
+        }
+        node.addEventListener('change', handler)
+        app.attachEventListener(node, 'change', handler, false, true, true)
       }
+
       if (isTextEditable(node)) {
         inputValues.set(id, node.value)
         sendInputValue(id, node)
+        const setFocus = () => {
+          Object.assign(node, { or_focusStart: +new Date() })
+        }
+        const inputEvent = (e: InputEvent) => {
+          const value = (e.target as HTMLInputElement).value
+          if (inputValues.get(id) === '' && value !== '') {
+            const inputTime = +new Date()
+            // @ts-ignore
+            const hesitationTime = inputTime - node.or_focusStart
+            Object.assign(node, { or_inputHesitation: hesitationTime })
+          }
+          inputValues.set(id, value)
+          debouncedUpdate(id, node)
+        }
+        const changeEvent = (e: InputEvent) => {
+          const value = (e.target as HTMLInputElement).value
+          if (inputValues.get(id) !== value) {
```

Comment:
it will be always `false` because `input` event happens instantly and updates it to the latest